applications/search/src/api/referenceData.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+import { getConfig } from '../config';
+
+export const referenceDataUrlBase = () => `${getConfig().referenceDataApi.host}/reference-data/`;
```

Comment:
This does not need to be a function.

Trailing slash?

Reply:
er det bedre med path som begynner med slash? eller kanskje, query som har base + / + path?